doc/profile_changes_SLE15.md

```diff
@@ -0,0 +1,473 @@
+
+### GPT is now the default partition type
+
+On x86_64 systems, GPT is now the preferred partition type. However, if you
+would like to retain the old behaviour, you could explictly indicate this in
+the profile setting the `disklabel` element to `msdos`.
+
+### Reading an existing /etc/fstab filesystem is not supported anymore
+
+For the time being, the ability to read an existing /etc/fstab from a previous
+installation when trying to determine the partitioning layout is not supported
+anymore.
+
+
+## Software
+
+The SLE15 installation medium contains only a very minimal set of packages
+to install. This minimal set does not include any server applications
+or advanced tools.
+
+If you need to install more packages then you need to use additional software
+repositories:
+
+- A registration server (the SUSE Customer Center or a SMT/RMT proxy)
+- Additional Packages DVD medium with SLE15 modules and extensions. The DVD
+  can be shared on the network via a local installation server.
+
+*Note: Using the registration server will grant the access to the maintenance
+updates. Maintenance updates are not available when using the DVD medium
+without registration.*
+
+### Using Modules or Extensions from the Registration Server
+
+If you want to add a module or extension from the registration server
+then add `addons` section to the registration configuration:
+
+```xml
+<suse_register>
+  <addons config:type="list">
+    <addon>
+      <name>sle-module-basesystem</name>
+      <version>15</version>
+      <arch>x86_64</arch>
+    </addon>
+  </addons>
+</suse_register>
+```
+
+For extensions which require a registration code write it into the `<reg_code>`
+tag for the respective extension. See more details in the [Module and Extension
+Dependencies](#module-and-extension-dependencies) section.
+
+### Using the Packages DVD Medium
+
+For using a physical Packages DVD medium use this XML snippet:
+
+```xml
+<add-on>
+  <add_on_products config:type="list">
+    <listentry>
+      <media_url><![CDATA[dvd:///]]></media_url>
+      <product>sle-module-basesystem</product>
+      <product_dir>/Module-Basesystem</product_dir>
+    </listentry>
+  </add_on_products>
+</add-on>
+```
+
+*Note: The `product` name must match the internal product name contained in the
+repository. If the product name does not match at installation AutoYaST
+will report an error.*
+
+If you have multiple physical DVD drives you can select a specific device
+using `devices` parameter in the URL, e.g. `dvd:///?devices=/dev/sr1`.
+
+### Using the Packages Medium from a Local Server
+
+You can share the DVD content on the local network via a NFS, FTP or HTTP server.
+
+In that case use the same XML snippet as above, just edit the `media_url`
+tag so it points to root of the medium on the server.
+
+
+### Renamed Software Patterns
+
+The software patterns have been also changed in SLE15. Some patterns have been
+renamed, a short summary is in the following table.
```

Comment:
Let's add something like: "Please, check carefully if all required packages are available in defined patterns and adjust profiles accordingly. Additionally, patterns and software has to be available in activated modules."

Reply:
:heavy_check_mark: Added below